src/Psl/Iter/drop_while.php

```diff
@@ -28,7 +25,7 @@
  */
 function drop_while(iterable $iterable, callable $predicate): Iterator
 {
-    return Internal\lazy_iterator(static function () use ($iterable, $predicate) {
+    return Iterator::from(static function () use ($iterable, $predicate) {
```

Comment:
please add missing return type ( there's couple others that are missing return type as well )

```suggestion
    return Iterator::from(static function () use ($iterable, $predicate): Generator {
```